cfn-resources/cloud-backup-snapshot-export-job/test/policy.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
Do we really need these many permissions in this role?

Reply:
yes we need this file